tgui/packages/tgui/interfaces/Radar.tsx

```diff
@@ -0,0 +1,145 @@
+  arrowstyle: string;
+  rot: number;
+  pointer: string;
+  color: string;
+  locy: number;
+  locx: number;
+};
+
+export const Radar = (props, context) => {
+  return (
+    <Window width={800} height={600} theme="ntos">
+      <Window.Content scrollable>
+        <RadarContent />
+      </Window.Content>
+    </Window>
+  );
+};
+
+export const RadarContent = (props, context) => {
+  return (
+    <Stack fill>
+      <Stack.Item position="relative" width={20.5}>
+        <ObjectDisplay />
+      </Stack.Item>
+      <Stack.Item
+        style={{
+          'background-image':
+            'url("' + resolveAsset('ntosradarbackground.png') + '")',
+          'background-position': 'center',
+          'background-repeat': 'no-repeat',
+          'top': '20px',
+        }}
+        position="relative"
+        m={1.5}
+        width={45}
+        height={45}>
+        <TargetDisplay />
+      </Stack.Item>
+    </Stack>
+  );
+};
+
+/** Returns object information */
+const ObjectDisplay = (props, context) => {
+  const { act, data } = useBackend<Data>(context);
+  const { object = [], scanning, selected } = data;
+
+  return (
+    <Section>
+      <Button
+        icon="redo-alt"
+        content={scanning ? 'Scanning...' : 'Scan'}
+        color="blue"
+        disabled={scanning}
+        onClick={() => act('scan')}
+      />
+      {!object.length && !scanning && <div>No trackable signals found</div>}
+      {!scanning &&
+        object.map((object) => (
+          <div
+            key={object.dev}
+            title={object.name}
+            className={classes([
+              'Button',
+              'Button--fluid',
+              'Button--color--transparent',
+              'Button--ellipsis',
+              object.ref === selected && 'Button--selected',
+            ])}
+            onClick={() => {
+              act('selecttarget', {
+                ref: object.ref,
+              });
+            }}>
+            {object.name}
+          </div>
+        ))}
+    </Section>
+  );
+};
+
+/** Returns target information */
+const TargetDisplay = (props, context) => {
+  const { data } = useBackend<Data>(context);
+  const { selected, target } = data;
+
+  if (!selected || !target) {
+    return null;
+  }
+  if (!Object.keys(target).length && !!selected) {
+    return (
+      <NoticeBox
+        position="absolute"
+        top={20.6}
+        left={1.35}
+        width={42}
+        fontSize="30px"
+        textAlign="center">
+        Signal Lost
+      </NoticeBox>
+    );
+  }
+  return target.userot ? (
+    <Box
+      as="img"
+      src={resolveAsset(target.arrowstyle)}
+      position="absolute"
+      top="20px"
+      left="243px"
+      style={{
+        'transform': `rotate(${target.rot}deg)`,
+      }}
+    />
+  ) : (
+    <Icon
+      name={target.pointer}
+      position="absolute"
+      size={2}
+      color={target.color}
+      top={target.locy * 10 + 19 + 'px'}
+      left={target.locx * 10 + 16 + 'px'}
```

Comment:
```suggestion
      left={(target.locx -1) * 10 + 16 + 'px'}
```
![dreamseeker_VjFhEpxNnP](https://user-images.githubusercontent.com/55142896/211219704-0afc098b-1705-46ec-9223-987a6b38803e.png)
![dreamseeker_J36eYXfalC](https://user-images.githubusercontent.com/55142896/211219708-4806bfa6-94d2-42e0-84ed-b274ca10e42f.png)
okay this is TERRIBLE but otherwise the interface is off by 1... there's probably a better place to do this, mind